platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -262,6 +270,17 @@ public void run() {
               if (observer != null) {
                 observer.onStatusChanged(status);
               }
+              try {
+                if (status.getDownloadState() == STATE_ACTIVE
+                        && status.getCompletedResourceSize() == 0) {
+                  telemetry.onOfflineDownloadStart(definition);
```

Comment:
This will only track the first time an offline region begins downloading. If it has been paused in a previous session, it may restart with a `>0` completed resource size, and skip a second _start_ event.